package.json

```diff
@@ -34,6 +34,7 @@
   "dependencies": {
     "@babel/core": "^7.4.5",
     "@babel/polyfill": "^7.4.4",
+    "eslint-plugin-react": "^7.13.0",
```

Comment:
Hello @matthenge ,
This package is already in devDependencies (line 32 of package.json). You can remove it to avoid redanduncy

Reply:
@Ogutu-Brian Done